test/msssim.jl

```diff
@@ -0,0 +1,73 @@
+using ImageFiltering
+
+@testset "MS-SSIM" begin
+    @info "test: MS-SSIM"
+
+    # constructors
+    iqi_a = MSSSIM()
+    iqi_b = MSSSIM(KernelFactors.gaussian(1.5, 11))
+    iqi_c = MSSSIM(KernelFactors.gaussian(1.5, 11), (0.0448, 0.2856, 0.3001, 0.2363, 0.1333))
+    iqi_d = MSSSIM(KernelFactors.gaussian(1.5, 11), (
+        (0.0448, 0.0448, 0.0448),
+        (0.2856, 0.2856, 0.2856),
+        (0.3001, 0.3001, 0.3001),
+        (0.2363, 0.2363, 0.2363),
+        (0.1333, 0.1333, 0.1333),
+    ))
+    @test (iqi_a.kernel == iqi_b.kernel == iqi_c.kernel == iqi_d.kernel) &&
+          (iqi_a.W == iqi_b.W == iqi_c.W == iqi_d.W)
+
+    # Gray test images
+    img1 = testimage("cameraman")
+    img2 = testimage("lena_gray_512")
+    @test assess_msssim(img1, img2) ≈ 0.0711 atol=1e-4
```

Comment:
Have you checked the results on other MSSSIM implementations? It would be great to get this matched.